### PR TITLE
DOCS-577 PGD-5.5-Release

### PR DESCRIPTION
**Pull request by djw-m**
A new release branch reverting the revert in EnterpriseDB/docs#5613

### CODE REVIEW

File: product_docs/docs/pgd/5/cli/installing/tpa.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: "Installing PGD CLI with Trusted Postgres Architect"
+---
+
+By default, Trusted Postgres Architect installs and configures PGD CLI on each PGD node. 
+
+If you want to install PGD CLI on any non-PGD instance in the cluster, attach the pgdcli role to that instance in Trusted Postgres Architect's configuration file before deploying. 
+
+See [Trusted Postgres Architect](/tpa/latest/) for more information.
+
+[Next: Using PGD CLI](../using_cli)
```
Comment (gvasquezvargas) [inline product_docs/docs/pgd/5/cli/installing/tpa.mdx:13]: The embeded "next" tile already points to that topic, so that seems superfluous here?

Comment (djw-m) [reply product_docs/docs/pgd/5/cli/installing/tpa.mdx:13]:
```suggestion
```

File: product_docs/docs/pgd/5/routing/administering.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## Switching the write leader
+
+Switching the write leader is a manual operation that you can perform to change the node that's the write leader. 
+It can be useful when you want to perform maintenance on the current write leader node or when you want to change the write leader for any other reason.
+When changing write leader, there are two modes, `strict` and `fast`. 
+In `strict` mode, the lag is checked before switching the write leader. It will wait until the lag is less than `route_writer_max_lag` before starting the switchover. This is the default.
+In `fast` mode, the write leader is switched immediately.
+You can also set a timeout parameter to specify the time to wait for the switchover to complete.
+
+!!! Note
```
Comment (gvasquezvargas) [inline product_docs/docs/pgd/5/routing/administering.mdx:15]: This note is being displayed as a code snippet. Maybe an additional space after the closing !!! will help? 

Screenshot 2024-05-16 at 11 35 38

**Review by gvasquezvargas — APPROVED** [review]
A note is being displayed as a code snippet, but apart from that LGTM!

**Pull request merged**